src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -1,19 +1,14 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-linux`
Can load a native assembly. Possitive test..Can load native dll
```powershell
Exception calling "LoadNative" with "0" argument(s): "Unable to load shared library '/tmp/tmpwHlG6x.tmp/linux-x64/libSkiaSharp.so' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: /tmp/tmpwHlG6x.tmp/linux-x64/libSkiaSharp.so: cannot open shared object file: No such file or directory"
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 51
51:         [TestNativeClass]::LoadNative() | Should -Not -BeNullOrEmpty

```
Can load a native assembly.Can load native dll
```powershell
Exception calling "LoadNative" with "0" argument(s): "Unable to load shared library '/tmp/tmpvjAVlM.tmp/linux-x64/nativedll.so' or one of its dependencies. In order to help diagnose loading problems, consider setting the LD_DEBUG environment variable: /tmp/tmpvjAVlM.tmp/linux-x64/nativedll.so: cannot open shared object file: No such file or directory"
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Basic/Assembly.LoadNative.Tests.ps1: line 116
116:         [TestNativeClass2]::LoadNative() | Should -Not -BeNullOrEmpty

```